### PR TITLE
Switch from Twitter to X

### PR DESCRIPTION
**Pull request by gsmet**
And fix a few unrelated things.

** If you are updating a guide, please submit your pull request to the main repository: https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc **

### CODE REVIEW

File: _includes/socials.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
     <div class="width-3-12 width-0-12-m"></div>
     <div class="width-3-12 width-0-12-m"></div>
     <div class="width-6-12 width-12-12-m">
-      <a href="http://www.twitter.com/@quarkusio" class="button-cta"><i class="fab fa-twitter"></i>Follow us on Twitter</a>
+      <a href="https://x.com/@quarkusio" class="button-cta"><i class="fab fa-twitter"></i>Follow us on X</a>
```
Comment (gsmet) [inline _includes/socials.html:20]: Unfortunately, FA 5 doesn't support X so we would have to move to FA 6.
Not sure the update will be straightforward and we need to make sure we can still use it for free.

**Review by gsmet — COMMENTED** [review]
@insectengine could you have a look at the comments?

**Comment by gsmet** [comment]
I merged this first step, I will let @insectengine address the comments above when he has some time.

**Comment by insectengine** [comment]
Thanks... looking into the switch to FA 6 to remove the need of images and pruning unused includes.

**Pull request merged**